docs/quick-start/index.md

```diff
@@ -0,0 +1,95 @@
+---
+sidebar_position: 0
+title: Quick Start V2
+description: Get started with Miden by installing Miden tools using the `midenup` toolchain, creating your first wallet, performing basic operations, and building your first smart contract!
+---
+
+# Quick Start
+
+Welcome to Miden! This guide will get you up and running with the Miden blockchain by walking you through the essential setup and core operations.
+
+## What is Miden?
+
+Miden is a privacy-focused, STARK-based blockchain that uses an account model where each account is a smart contract. Unlike traditional blockchases where accounts simply hold balances, Miden accounts are programmable entities that can execute custom logic, store data, and manage assets autonomously.
+
+Key concepts you'll encounter:
+
+- **Accounts**: Smart contracts that hold assets and execute code
+- **Notes**: Private data transfers between accounts
```

Comment:
Notes aren't always private, so writing private here, to me, suggests they are always private.

Nit: I think we sometimes also referred to notes as the messages between the actors (accounts), maybe that's an angle here, e.g. "Messages that exchange data and assets between accounts" or something along those lines.